From 12b0d936611617ac9f6ae79bbb4acd54482fda11 Mon Sep 17 00:00:00 2001
From: Pardha Saradhi K <pardha.saradhi.kesapragada@intel.com>
Date: Mon, 8 Aug 2016 11:37:54 +0530
Subject: [PATCH 4694/4706] ASoC: Intel: Skylake: Fix compilation issues for
 debugfs

commit 583d00479404e7f1d5ab25fb0b36534764ea8bf9 from
git://git.yoctoproject.org/linux-yocto-4.1

This patch fixes compilation issues that arise when debugfs
is removed from menuconfig by selecting CONFIG_DEBUG_FS=n

Change-Id: Iea96d82d5069b256606d2146d7185d7837866868
Signed-off-by: Pardha Saradhi K <pardha.saradhi.kesapragada@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/10803
Reviewed-by: B, Jayachandran <jayachandran.b@intel.com>
Reviewed-by: Babu, Ramesh <ramesh.babu@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Tested-by: Jayanti, Satya Charitardha <satya.charitardha.jayanti@intel.com>
---
 sound/soc/intel/skylake/skl.h |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/sound/soc/intel/skylake/skl.h b/sound/soc/intel/skylake/skl.h
index efbf05a..532a83f 100644
--- a/sound/soc/intel/skylake/skl.h
+++ b/sound/soc/intel/skylake/skl.h
@@ -162,29 +162,29 @@ void skl_update_dsp_debug_info(struct skl_debug *d,
 #else
 
 struct skl_debug {
-}
+};
 
-struct skl_debug *skl_debugfs_init(struct skl *skl)
+static inline struct skl_debug *skl_debugfs_init(struct skl *skl)
 {
 	return NULL;
 }
-void skl_debugfs_exit(struct skl_debug *d)
+static inline void skl_debugfs_exit(struct skl_debug *d)
 {
 }
 
-struct nhlt_specific_cfg
+static inline struct nhlt_specific_cfg
 *skl_nhlt_get_debugfs_blob(struct skl_debug *d, u8 link_type, u32 instance, u8 stream)
 {
 	return NULL;
 }
 
-void skl_debug_init_module(struct skl_debug *d,
+static inline void skl_debug_init_module(struct skl_debug *d,
 			struct snd_soc_dapm_widget *w,
 			struct skl_module_cfg *mconfig)
 {
 }
 
-void skl_update_dsp_debug_info(struct skl_debug *d,
+static inline void skl_update_dsp_debug_info(struct skl_debug *d,
 		struct platform_info *dbg_info)
 {
 }
-- 
1.7.5.4

